packages/ic-management/candid/ic-management.d.ts

```diff
@@ -162,18 +174,42 @@ export interface _SERVICE {
     ],
     http_response
   >;
+  install_chunked_code: ActorMethod<
```

Comment:
Are you going to provide an addition PR to support this new feature (and the two other related functions)? It might be interesting for devs.

Reply:
Yes, I want to do it in another PR because it needs some minor design decision that might add breaking change or not.